nl/tested-judge/exercise-config/README.md

```diff
@@ -4,21 +4,32 @@ description: "Configuratie TESTed"
 sidebarDepth: 2
 ---
 
-# Configuratie TESTed
-De TESTed judge heeft een verplicht veld `evaluation.plan_name` in het `config.json` bestand van een Dodona oefening.
-Dit veld moet de naam van een testplan, in de map `evaluation` van de oefening, bevatten.
-Het testplan kan zowel een [DSL-testplan](../dsl) zijn, als een [geavanceerd testplan](../json).
-De DSL-testplannen moet eindigen op één van de YAML-extensies (`.yaml` of `.yml`).
-Voor de geavanceerde testplannen zijn er geen beperkingen op de bestandsextensie, behalve de YAML-extensies.
-
-Naast het verplichte testplan-veld, heeft TESTed enkele configuratieopties die ingesteld kunnen worden.
-De opties moeten ingesteld worden in het `config.json` van een Dodona oefening
-(zie [Oefeningconfiguratie](../../references/exercise-config)).
-De opties moeten meegegeven worden in het object `evaluation.options`.
-
-## JSON-Schema
-Hieronder kun je het JSON Schema vinden van de verschillende opties.
-De opties zullen in de volgende paragrafen verder uitgelegd worden.
+# TESTed configureren
+
+Als judge die oplossingen voor programmeeroefeningen automatisch beoordeelt, 
+heeft TESTed een verplicht veld `evaluation.plan_name` in het 
+configuratiebestand van de oefening (`config.json`). Dit veld moet aangeven
+wat de bestandsnaam is van het testplan in de map `evaluation` van de oefening.
+
+Voor testplannen zijn er twee opties. TESTed ondersteunt een domeinspecifieke
+taal (DSL; *domain specific language*) om de testen te beschrijven waaraan een
+ingediende oplossing zal onderworpen worden. Het gebruik van deze DSL is de 
+meest eenvoudige manier om de testen te beschrijven. De specificatie van 
+[**DSL-testplannen**](../dsl) gebeurt in het 
+[YAML-formaat](https://en.wikipedia.org/wiki/YAML){: target="_blank"} in 
```

Comment:
@pdawyndt de target blanks zijn niet nodig. Externe links worden automatisch herkend, voorzien van een "extern" icoontje en in een nieuwe tab geopend door de website software.

Reply:
solved